readme.md

```diff
@@ -115,7 +120,7 @@ Write some basic CSS that demonstrates changing a CSS property when the device w
 How is rebase different than a merge?
 
 Your Answer:
-```text
+```rebase smoothes out past commit tracks but this can be dangerous because it removes access to past versions.
 
 ```
```

Comment:
:+1: for rebase. Merge accomplishes the same goal but does not modify git history. It just accumulates all the new changes into a single commit.